<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>soccer_window</class>
 <widget class="QWidget" name="soccer_window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Play Soccer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Untitled1.png</normaloff>Untitled1.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="game_view">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>701</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-color: #000;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_game">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="switch_fields">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>70</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch ATK/DEF Area</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CPH">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CPH</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="fps_lcd">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>550</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-color: #000;</string>
   </property>
  </widget>
  <widget class="QLabel" name="fps_label">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>550</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FPS:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="battery_level_group">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Battery Levels:</string>
   </property>
   <widget class="QProgressBar" name="gandalf_battery_level">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="leona_battery_level">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="presto_baterry_level">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="robots_roles_group">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>120</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Robots Roles:</string>
   </property>
   <widget class="QLabel" name="gandalf_role_label">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>     -- None --</string>
    </property>
   </widget>
   <widget class="QLabel" name="leona_role_label">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>     -- None --</string>
    </property>
   </widget>
   <widget class="QLabel" name="presto_role_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>    -- None --</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="gandalf_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Gandalf:</string>
   </property>
  </widget>
  <widget class="QLabel" name="leona_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Leona:</string>
   </property>
  </widget>
  <widget class="QLabel" name="presto_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Presto:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="optionsGroupBox">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>290</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Options:</string>
   </property>
   <widget class="QCheckBox" name="show_field_areas_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Areas</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_rnames_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Names</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_rcentroids_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Centroids</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_visionlogs_checkbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Vision Logs</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="read_parameters">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>130</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Read Parameters</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="detection_status_group">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>480</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Detection:</string>
   </property>
   <widget class="QLabel" name="gandalf_detec_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Detected</string>
    </property>
   </widget>
   <widget class="QLabel" name="leona_detec_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Detected</string>
    </property>
   </widget>
   <widget class="QLabel" name="presto_detec_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Not Detected</string>
    </property>
   </widget>
   <widget class="QLabel" name="gandalf_detec_col_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leona_detec_col_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="presto_detec_col_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gandalf_loss_rate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leona_loss_rate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="presto_loss_rate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="ball_detec_label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>430</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ball not found</string>
   </property>
  </widget>
  <widget class="QLabel" name="ball_detec_col_label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>430</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : red</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cam_id_label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>250</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Cam ID:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="cam_id_spinBox">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>250</y>
     <width>47</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="serial_status_group">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Serial Status:</string>
   </property>
   <widget class="QLabel" name="gandalf_serial_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Closed</string>
    </property>
   </widget>
   <widget class="QLabel" name="leona_serial_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Closed</string>
    </property>
   </widget>
   <widget class="QLabel" name="presto_serial_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Closed</string>
    </property>
   </widget>
   <widget class="QLabel" name="gandalf_serial_col">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="leona_detec_col">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="presto_detec_col">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : red</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>460</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <zorder>optionsGroupBox</zorder>
  <zorder>game_view</zorder>
  <zorder>start_game</zorder>
  <zorder>switch_fields</zorder>
  <zorder>CPH</zorder>
  <zorder>fps_lcd</zorder>
  <zorder>fps_label</zorder>
  <zorder>battery_level_group</zorder>
  <zorder>robots_roles_group</zorder>
  <zorder>gandalf_label</zorder>
  <zorder>leona_label</zorder>
  <zorder>presto_label</zorder>
  <zorder>read_parameters</zorder>
  <zorder>detection_status_group</zorder>
  <zorder>ball_detec_label</zorder>
  <zorder>ball_detec_col_label</zorder>
  <zorder>cam_id_label</zorder>
  <zorder>cam_id_spinBox</zorder>
  <zorder>serial_status_group</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
